packages/next/src/build/compile-config.ts

```diff
@@ -0,0 +1,28 @@
+import { readFileSync, writeFileSync, mkdirSync } from 'fs'
+import { join } from 'path'
+import { transformSync } from './swc'
+
+export function compileConfig({
+  configPath,
+  cwd,
+}: {
+  configPath: string
+  cwd: string
+}): string {
+  const content = readFileSync(configPath, 'utf-8')
+  const compiledFilePath = join(cwd, '.next', 'next.config.mjs')
+
+  const compiled = transformSync(content, {
+    jsc: {
+      target: 'es5',
```

Comment:
now that next requires node 18 >=, i think the target could be `es2022` right?

Reply:
Yeah, you're right. Since next.config also takes async functions as values, we need to modify the target. Thanks!